AssemblyUnhollower/Passes/Pass20GenerateStaticConstructors.cs

```diff
@@ -149,8 +150,8 @@ private static void EmitLoadTypeNameString(this ILProcessor ctorBuilder, Assembl
                 ctorBuilder.Emit(OpCodes.Ldstr, originalTypeReference.FullName);
             else
             {
-                ctorBuilder.Emit(newTypeReference.IsByReference ? OpCodes.Ldc_I4_1 :  OpCodes.Ldc_I4_0);
-                ctorBuilder.Emit(OpCodes.Call, imports.Module.ImportReference(new GenericInstanceMethod(imports.Il2CppRenderTypeNameGeneric) {GenericArguments = {newTypeReference}}));
+                ctorBuilder.Emit(newTypeReference.IsByReference ? OpCodes.Ldc_I4_1 : OpCodes.Ldc_I4_0);
```

Comment:
This would be only used for methods that somehow don't get a token (in fact, this should have been removed a long time ago as in practice everything has tokens). It would be better to actually investigate those cases when there are tokenless methods and fix those.

Reply:
This is the case of:
`IL2CPP.RenderTypeName<someType*>(true)` - .net doesn’t like class pointers as a generic type input.